### ISSUE TITLE
Opening a `.excalidraw` file clobbers all ad hoc drawings that weren't saved to a file

### ISSUE THREAD

**Issue by some-ordinary-dev** [excalidraw/excalidraw#7800]
# Problem

If a user has excalidraw open with ad hoc drawings (not saved to disk, only stored in browser local storage) and opens an `excalidraw` file via a file explorer a new instance of excalidraw will be opened (if installed via a PWA) and will clobber all stored ad hoc drawings without warning the user.

# Expected Behavior

Excalidraw should warn the user that any ad hoc drawings will be lost if they open a file, and/or allow them to save their ad hoc drawings to disk. If the user had a file from disk open when they attempt to open another file excalidraw should prompt them to save their changes to disk or automatically save the changes to disk for them.

Another option would be to store multiple sessions in browser storage so that all ad hoc drawings don't have to be saved to disk but would be persisted in browser storage.

**Comment by amitupadhyaymh** [comment]
yeah.. exactly what happened to me. lost all the drawings. any way to recover?

**Comment by ezradyck** [comment]
> yeah.. exactly what happened to me. lost all the drawings. any way to recover?

I didn't find any way to get the drawings back.
